runelite-client/src/test/java/net/runelite/client/plugins/chatfilter/ChatFilterPluginTest.java

```diff
@@ -64,6 +69,8 @@
 	@Inject
 	private ChatFilterPlugin chatFilterPlugin;
 
+	private static final String CENSOR_MESSAGE_STRING = "Hey, everyone, I just tried to say something very silly!";
```

Comment:
Re-use the censor message from plugin. (make it package-private and `@VisibleForTesting`)